syntax = "proto3";

import "super.proto";

service ShowTime {
    rpc GetMovieByDate(MovieDate) returns (MovieID) {}
    rpc GetSchedule(Empty) returns (stream TimeShow) {}
}

message MovieDate {
    string date = 1;
}

message MovieID {
    repeated string movies = 1;
}